### PR TITLE
New Major release 0.10.0

### PR DESCRIPTION
**Pull request by OnnoEbbens**
including the new functionality to read data from Lizard

### CODE REVIEW

File: hydropandas/io/lizard.py [hunk 1]
```diff
@@ -0,0 +1,689 @@
...
+        elif metadata.get("start_hand") is None:
+            metadata["timeseries_type"] = "diver"
+        elif metadata.get("start_diver") is None:
+            metadata["timeseries_type"] = "hand"
+
+    return metadata
+
+
+def get_timeseries_uuid(uuid, code, tube_nr, tmin, tmax, page_size=100000):
+    """
```
Comment (dbrakenhoff) [inline hydropandas/io/lizard.py:275]: code and tube_nr are not used in this function

File: hydropandas/io/lizard.py [hunk 2]
```diff
@@ -0,0 +1,689 @@
...
+        ObsCollection DataFrame with the 'obs' column
+
+    """
+
+    if isinstance(extent, (list, tuple)):
+        polygon_T = extent_to_wgs84_polygon(extent)
+
+    elif isinstance(extent, str) or isinstance(extent, pathlib.PurePath):
+        polygon = geopandas.read_file(extent)
+        polygon_T = polygon.to_crs("WGS84", "EPSG:28992").loc[0, "geometry"]
```
Comment (dbrakenhoff) [inline hydropandas/io/lizard.py:653]: Is this transformation correct, given some of the issues with EPSG 28992 we've encountered before?

**Review by dbrakenhoff — COMMENTED** [review]
Nice work on the Lizard stuff! 

I added a few notes for the lizard functions to check into later. 
- vitens API is currently hardcoded and the only one we officially support right now
- the EPSG:28992 to WGS84 transformation should be checked at some point.

For the rest I ran docformatter and fixed some typos here and there.

**Review by dbrakenhoff — APPROVED** [review]

**Comment by OnnoEbbens** [comment]
Failing tests are not due to PR, see #183

**Pull request merged**